lib/engines/hcktest/hcktest.rb

```diff
@@ -118,7 +118,16 @@ def target
 
     def self.platform(logger, options)
       platform_name = options.test.platform
-      Json.read_json("#{PLATFORMS_JSON_DIR}/#{platform_name}.json", logger)
+      platform = Json.read_json("#{PLATFORMS_JSON_DIR}/#{platform_name}.json", logger)
+
+      if options.test.svvp
+        svvp_info = Models::SVVPConfig.from_json_file(SVVP_JSON, logger)
+
+        platform['viommu_state'] ||= svvp_info.viommu_state
+        logger.debug("Platform updated from SVVP config: #{platform}")
```

Comment:
I don't think this log is useful unless you are modifying this particular method.